src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import './Todo.css';
+
+const TodoForm = (props) => {
+  return (
+    <form className="todo-form" onSubmit={props.addTodo}>
+      <div className="form-wrapper">
+        <input
+          type="text"
+          value={props.todoInput}
+          name="todoInput"
+          onChange={props.handleChanges}
+          placeholder="What needs to be done?"
+          autoComplete="off"
+          required
```

Comment:
I can tell you played around with input attributes, the autoComplete here isn't exactly necessary but will come in handy for future, more complicated projects, but `required` is incredibly useful and abstracts away a lot of conditional code in your `addTodo` function, well done